src/components/QRCodeWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { IS_EMBEDDED_APP } from "~utils/embedded/embedded.constants";
+
+export const QRCodeWrapper = styled.div<{ size?: number }>`
```

Comment:
Unless `QRCodeLoop` is used somewhere without `QRCodeWrapper`, I think `QRCodeWrapper` should just be part of `QRCodeLoop`.